core/src/main/kotlin/model/Documentable.kt

```diff
@@ -57,8 +62,13 @@ class Property(
     override val receiver: Parameter?,
     override val expected: PlatformInfo?,
     override val actual: List<PlatformInfo>,
-    override val extra: MutableSet<Extra> = mutableSetOf()
-) : CallableNode() {
+    override val extra: MutableSet<Extra> = mutableSetOf(),
+    val type: TypeWrapper?,
```

Comment:
What if the type is different on different platforms? Eg. `class B:A`  and `expected val x : A` in common and `actual val x: B` in jvm (credit @Kordyjan)

Reply:
Removed type info.